### PR TITLE
Replace explicit z-level checks with defines

### PR DESCRIPTION
**Pull request by SpaceManiac**
:cl:
code: Explicit z-level checks have been changed to use defines.
/:cl:

Intended to get some of the "touches every file" of #32037 out of the way without yet changing any functionality. `z == ZLEVEL_` and `z in GLOB.station_z_levels` have all been removed. Some uses of these in list context or in the mapping module remain, to be dealt with at a later time.

### CODE REVIEW

**Comment by Cyberboss** [comment]
This is good for bticoinc

**Comment by ShizCalev** [comment]
Might want to rename z_is_planet

**Comment by ShizCalev** [comment]
Conflict.

**Comment by SpaceManiac** [comment]
Conflict resolved.

File: code/game/atoms.dm [hunk 1]
```diff
@@ -139,10 +139,10 @@
 	if(!T)
 		return 0
```
Comment (ShizCalev) [inline code/game/atoms.dm]: FALSE

File: code/game/atoms.dm [hunk 2]
```diff
@@ -139,10 +139,10 @@
 	if(!T)
 		return 0
 
-	if(T.z != ZLEVEL_CENTCOM)//if not, don't bother
+	if(!is_centcom_level(T.z))//if not, don't bother
 		return 0
```
Comment (ShizCalev) [inline code/game/atoms.dm]: FALSE

File: code/game/atoms.dm [hunk 3]
```diff
@@ -139,10 +139,10 @@
 	if(!T)
 		return 0
 
-	if(T.z != ZLEVEL_CENTCOM)//if not, don't bother
+	if(!is_centcom_level(T.z))//if not, don't bother
 		return 0
 
-	if(istype(T.loc, /area/shuttle/syndicate) || istype(T.loc, /area/syndicate_mothership))
+	if(istype(T.loc, /area/shuttle/syndicate) || istype(T.loc, /area/syndicate_mothership) || istype(T.loc, /area/shuttle/assault_pod))
 		return 1
```
Comment (ShizCalev) [inline code/game/atoms.dm]: TRUE

File: code/game/atoms.dm [hunk 4]
```diff
@@ -139,10 +139,10 @@
...
 
-	if(T.z != ZLEVEL_CENTCOM)//if not, don't bother
+	if(!is_centcom_level(T.z))//if not, don't bother
 		return 0
 
-	if(istype(T.loc, /area/shuttle/syndicate) || istype(T.loc, /area/syndicate_mothership))
+	if(istype(T.loc, /area/shuttle/syndicate) || istype(T.loc, /area/syndicate_mothership) || istype(T.loc, /area/shuttle/assault_pod))
 		return 1
 
 	return 0
```
Comment (ShizCalev) [inline code/game/atoms.dm]: FALSE

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**